lib/models/living.py

```diff
@@ -0,0 +1,48 @@
+from dataclasses import dataclass
+from typing import List, Dict
+
+from lib.models.character_class import CharacterClass
+from lib.models.entity import *
+from lib.models.enums import Ability, Skill, Alignment
+
+
+@dataclass
+class Living(Entity):
```

Comment:
I'd maybe change this to `LivingEntity` or something instead, so it's a bit clearer. Even `Creature`

Reply:
LivingEntity  is probably better, Creature just makes me think of monster/enemy.

Reply:
Are the undead living?

Reply:
[A solid answer ](https://rpg.stackexchange.com/questions/47254/what-is-the-definition-of-creature-and-is-it-used-consistently) that ways me towards Creature.